modules/ocm-nw-networkpolicy-create.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * networking/network_policy/creating-network-policy.adoc
+// * post_installation_configuration/network-configuration.adoc
+
+:name: network
+:role: admin
+ifeval::["{context}" == "configuring-multi-network-policy"]
+:multi:
+:name: multi-network
+:role: cluster-admin
+endif::[]
+
+:_content-type: PROCEDURE
+[id="ocm-nw-networkpolicy-create_{context}"]
+= Creating a {name} policy using {cluster-manager}
+
+To define granular rules describing ingress or egress network traffic allowed for namespaces in your cluster, you can create a {name} policy.
+
+.Prerequisites
+
+* You logged in to {cluster-manager-url}.
+* You created an {product-title} cluster.
+* You configured an identity provider for your cluster.
+* You added your user account to the configured identity provider.
+
+.Procedure
+
+. From {cluster-manager-url}, click on the cluster you want to access.
+
+. Click *Open console*.
+
+. Click on your identity provider and provide your credentials to log into the cluster.
+
+. Under *Networking*, click *NetworkPolicies*.
+
+. Click *Create NetworkPolicy*.
+
+. Provide a name for the policy in the "Policy name" field.
+
+. Optional: you can provide a specific pod's label and selector if this policy applies to only specific pod(s). If you do not select a specific pod, then this policy will be applicable to all pods on the cluster.
+
+. You can  block all ingress and egress traffic by using the "Deny all ingress traffic" or "Deny all egress traffic" checkboxes. You can also add any combination of ingress and egress rules, allowing you to specify the port, namespace, or IP blocks you want to approve.
+
+.Adding ingress rules
+. Select *Add ingress rule* to configure a new rule. This action creates a new *Ingress rule* row with a *Add allowed source* dropdown to specify how you want to limit inbound traffic.
```

Comment:
"specify how you want to limit inbound traffic" i wonder if this could be elaborated on a little more. As is, I am wondering how would I know what I want to limit to.

Disregard this comment if you don't think it is relevant. I am not that familiar with this content.